### ISSUE TITLE
Training Strategy

### ISSUE THREAD

**Issue by abhiagwl4262** [lxtGH/OctaveConv_pytorch#21]
Recently, I am doing experiments on octave convolution. Should I be getting the same performance as vanilla convolution neural network by keeping the learning strategy same while training my model with octave convolution or I might have to change my learning strategy in order to get the same performance as that of vanilla convolution neural network?

**Comment by lxtGH** [comment]
Sorry for late reply. Did you try with cosine learning rate? The orginal paper use cosine learning rate for training and please see the libs/lr_scheduler.py
